src/_pytest/cacheprovider.py

```diff
@@ -499,7 +499,11 @@ def pytest_addoption(parser: Parser) -> None:
         dest="last_failed_no_failures",
         choices=("all", "none"),
         default="all",
-        help="Which tests to run with no previously (known) failures",
+        help="Determines whether to execute tests when there are no previously (known)"
```

Comment:
Please update this with the above suggestion.

Reply:
update the docs here: 75e815f6c1dd5705c3a1f6d11b320013775b59e4